examples/dynamic/views/Html.js

```diff
@@ -2,9 +2,9 @@ var React = require('react');
 var ReactDOMServer = require('react-dom/server');
 var Content = require('./Content');
 
-module.exports = React.createClass({
+export default class Html extends React.Component {
```

Comment:
Please leave this as `module.exports` as we're still using commonjs modules for the rest of this. You may need to create the class first before assigning to `module.exports`.